src/terra-core/TerraCoreDataModel.ttl

```diff
@@ -2972,6 +2991,8 @@ TerraCore:usesReferenceAssembly
   rdfs:domain TerraCore:AlignmentFile ;
   rdfs:domain TerraCore:SequenceFile ;
   rdfs:domain TerraCore:SequencingActivity ;
+  rdfs:domain TerraCore:VariantCall ;
+  rdfs:domain TerraCore:VariantCallingActivity ;
   rdfs:label "usesReferenceAssembly" ;
   rdfs:range xsd:anyURI ;
```

Comment:
`  rdfs:range [
      owl:unionOf (
          xsd:anyURI
          xsd:string
        ) ;
    ] ;`

Reply:
next checkin fixes this